runtime/proto/common.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package org.corfudb.runtime;
+option java_package = "org.corfudb.runtime.proto";
```

Comment:
can we name this file as rpc_common.proto or similar instead of just common.proto?
https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Done